src/distribution.js

```diff
@@ -1061,4 +1061,45 @@ jStat.extend(jStat.triangular, {
   }
 });
 
+jStat.extend(jStat.laplace, {
+  pdf: function pdf(x, mu, b) {
+    return (b <= 0) ? 0 : (Math.exp(-Math.abs(x - mu) / b)) / (2 * b);
+  },
+
+  cdf: function cdf(x, mu, b) {
+    if (b <= 0) { return 0; }
+
+    if(x < mu) {
+      return 0.5 * Math.exp((x - mu) / b);
+    } else {
+      return 1 - 0.5 * Math.exp(- (x - mu) / b);
+    }
+  },
+
+  mean: function(mu, b) {
+    return mu;
+  },
+
+  median: function(mu, b) {
+    return mu;
+  },
+
+  mode: function(mu, b) {
+    return mu;
+  },
+
+  variance: function(mu, b) {
+    return 2 * b * b;
+  },
+
+  sample: function sample(mu, b) {
+    var u = Math.random() - 0.5;
+    var sign = function(x) {
+      return x / Math.abs(x);
+    };
```

Comment:
On modern JS engines this can't be optimized properly. It's better to hoist the function out so the optimizing compiler only has to compile the function once. Otherwise it has to compile the function on every execution.

I can take care of it here before landing the PR.